lib/watchman/server.ex

```diff
@@ -33,7 +33,7 @@ defmodule Watchman.Server do
   end
 
   defp parse_host(host) when is_binary(host) do
-    case host |> to_char_list |> :inet.parse_address do
+    case host |> to_charlist |> :inet.parse_address do
```

Comment:
Vim told me that `to_char_list` is deprecated. Changed it to `to_charlist`. 

The new version was introduced in `1.3.0`.